testing/utils.go

```diff
@@ -81,3 +83,12 @@ func UnmarshalMsgResponses(cdc codec.Codec, data []byte, msgs ...codec.ProtoMars
 
 	return nil
 }
+
```

Comment:
This was also added in another in-flight PR. There should be no merge conflicts, but I'll take care of them if they arise :)